include/constants.h

```diff
@@ -427,12 +427,28 @@
 
 /** @} */
 
-#define SPEC_MACRO_DEFINE "%define"
-#define SPEC_MACRO_GLOBAL "%global"
+#define SPEC_MACRO_DEFINE      "%define"
+#define SPEC_MACRO_GLOBAL      "%global"
 #define SPEC_SECTION_CHANGELOG "%changelog"
-#define SPEC_SECTION_FILES "%files"
-#define SPEC_TAG_RELEASE "Release:"
-#define SPEC_DISTTAG "%{?dist}"
+#define SPEC_SECTION_FILES     "%files"
+#define SPEC_TAG_RELEASE       "Release:"
+#define SPEC_DISTTAG           "%{?dist}"
+
+/*
+ * Names of rpmbuild(1) subdirectories.  rpminspect models an rpmbuild
+ * layout similar to how rpmbuild is used for distributions like
+ * Fedora Linux.  That is there is a top level directory (topdir) and
```

Comment:
That is there is reads a little bumpy; I might reword this to something like

In this example, there is...
In Fedora, there is...

Something like that.

Reply:
Oh yeah, that's a terrible comment.  Fix coming.

Reply:
```
/*                                                                                                                                                            
 * Names of rpmbuild(1) subdirectories.  rpminspect models an rpmbuild                                                                                        
 * layout similar to how rpmbuild is used for distributions like                                                                                              
 * Fedora Linux.  The main idea is a top level directory is defined                                                                                           
 * (topdir) and all of the other directories used by rpmbuild are                                                                                             
 * within topdir.  For the purposes of rpminspect, the topdir                                                                                                 
 * definition here is a subdirectory in the working directory location                                                                                        
 * rpminspect is using.                                                                                                                                       
 */
```

Eh?